NOTICE

```diff
@@ -11,8 +11,7 @@ License file: nbmolviz/static/3dmol/LICENSE
     D3.js
```

Comment:
@avirshup Does this NOTICE file look good like this by the way?  D3 is no longer a direct dependency, only indirectly via nbmolviz2d.  Should we list nbmolviz2d here instead of D3?